processor/k8sprocessor/processor.go

```diff
@@ -92,15 +93,18 @@ func (kp *kubernetesprocessor) ConsumeTraceData(ctx context.Context, td consumer
 	// check if the application, a collector/agent or a prior processor has already
 	// annotated the batch with IP.
 	if td.Resource != nil {
-		podIP = td.Resource.Labels[ipLabelName]
+		podIP = td.Resource.Labels[k8sIPLabelName]
+		if podIP == "" {
+			podIP = td.Resource.Labels[clientIPLabelName]
+		}
 	}
 
 	// Jaeger client libs tag the process with the process/resource IP and
 	// jaeger to OC translator maps jaeger process to OC node.
 	// TODO: Should jaeger translator map jaeger process to OC resource instead?
 	if podIP == "" && td.SourceFormat == sourceFormatJaeger {
 		if td.Node != nil {
-			podIP = td.Node.Attributes[ipLabelName]
+			podIP = td.Node.Attributes[clientIPLabelName]
```

Comment:
Is it possible that `k8sIPLabelName` exists in `td.Node.Attributes` at this point? Do we need to check for it as well?

Reply:
Unlikely but possible. I think it can happen if Otel agent detects and adds the IP to resource but then exports to Otel collector in Jaeger or SAPM instead of OTLP/OC. I don't see any downsides to covering for this case. Will update.